index.html

```diff
@@ -0,0 +1,359 @@
+        </div>
+      </div>
+    </header>
+    <section class="nav">
+      <div class="container">
+        <div class="row">
+          <div class="col-lg-2 col-md-12">
+            <p>Bhromaon</p>
+          </div>
+          <div class="col-lg-10 col-md-12">
+            <ul class="menu-main">
+              <li><a href="#" class="first_link">Who We Are</a></li>
+              <li><a href="#">What We Do</a></li>
+              <li><a href="#">Travel With Us</a></li>
+              <li><a href="#">Contact Us</a></li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="WhyChooseUs">
+      <div class="container">
+        <div class="row">
+          <div class="col-lg-6 col-md-12 reverse">
+            <div class="row">
+              <div class="col-12"><h2>Why Choose Us</h2></div>
+              <div class="col-12">
+                <p>
+                  Tion and dislike men who are so beguiled and demoralized by
+                  desire by the charms of pleasure of the moment, so blinded by
+                  desire, that they cannot foresee Ddislike men who are so
+                  beguiled and demoralized by the charms of pleasure of the
+                  moment, so blinded by desire, that.
+                </p>
+              </div>
+              <div class="col-12 WhyChooseUs_button">
+                <a href="#" class="button">Explore Places </a>
+              </div>
+            </div>
+          </div>
+          <div class="col-6 col-lg-6 col-sm-12 morph">
+            <div class="row">
+              <div class="col-lg-12 col-sm-6 first">
+                <img class="first" src="img/Image2.png" alt="house" />
+              </div>
+              <div class="col-lg-12 col-sm-6 second">
+                <img class="second" src="img/Image1.png" alt="mountains" />
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="SearchPlaces">
+      <div class="container">
+        <div class="row">
+          <div class="col-4">
+            <div class="row">
+              <div class="col-12"><span class="icon-search"></span></div>
+              <div class="col-12"><p>Search Places</p></div>
+            </div>
+          </div>
+          <div class="col-4">
+            <div class="row">
+              <div class="col-12"><span class="icon-check-circle"></span></div>
+              <div class="col-12"><p>Select Places</p></div>
+            </div>
+          </div>
+          <div class="col-4">
+            <div class="row">
+              <div class="col-12"><span class="icon-thumbs-up"></span></div>
+              <div class="col-12"><p>Book & Enjoy</p></div>
+            </div>
+          </div>
+          <object
+            class="dotted_line"
+            data="img/Line.svg"
+            type="image/svg+xml"
+          ></object>
```

Comment:
Можно использовать просто **img**
```suggestion
          <img src="img/Line.svg" alt="dotted line">
```